.sphinx/_templates/footer.html

```diff
@@ -101,5 +101,21 @@
 
 
     </div>
+    <div>
+      {% if github_url and github_folder and pagename and page_source_suffix %}
+          {% set contributors = get_contribs(github_url, github_folder, pagename, page_source_suffix) %}
+          {% if contributors %}
+            <a href="#" class="display-contributors">Thank you to the {{ contributors |length }} contributors!</a>
```

Comment:
```suggestion
            <a class="display-contributors">Thank you to the {{ contributors |length }} contributors!</a>
```
On a long page (like the ReadMe), the `href="#"` will make the page scroll to the top, which is annoying. I think it should work without.

Reply:
Yeah, thank you for this.